src/__tests__/app/create-text-message-template/__snapshots__/page.test.tsx.snap

```diff
@@ -1,12 +1,14 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`CreateSmsTemplatePage 1`] = `
-<DocumentFragment>
-  <h2
-    class="nhsuk-heading-l"
-    data-testid="page-sub-heading"
-  >
-    Placeholder SMS page
-  </h2>
-</DocumentFragment>
+exports[`CreateSmsTemplatePage should render CreateSmsTemplate component when session is found 1`] = `
```

Comment:
Just on the name of the file`create-text-message-template` - do you think we should go for `create-sms-message-template` to be consistent?

Reply:
Ignore, just realised this is the path for the page.